postgres/src/test/java/org/jdbi/v3/postgres/TestSqlArrays.java

```diff
@@ -115,12 +140,24 @@ public void testIntList() throws Exception {
         void insertUuidArray(UUID[] uuids);
 
         @SqlQuery(U_SELECT)
-        List<UUID> fetchUuidList();
+        // Wrap in an Optional container, so SQL object knows that the row type is List<UUID>, not UUID
+        Optional<List<UUID>> fetchUuidList();
+
+        @SqlQuery(U_SELECT)
+        // Wrap in an Optional container, so SQL object knows that the row type is List<UUID>, not UUID
+        Optional<ArrayList<UUID>> fetchUuidArrayList();
+
+        @SqlQuery(U_SELECT)
+        // Wrap in an Optional container, so SQL object knows that the row type is List<UUID>, not UUID
```

Comment:
Would it be more readable to use an annotation, perhaps bringing back `@SingleValueResult` or something similar?